afm/logging.py

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright 2020 IBM Corp.
+# SPDX-License-Identifier: Apache-2.0
+#
+import logging
+
+FORMATS = {}
+
+class FybrikFormatter(logging.Formatter):
+    def format(self, record):
+        log_fmt = FORMATS.get(record.levelno)
+        formatter = logging.Formatter(log_fmt, "%H:%M%p")
+        return formatter.format(record)
+
+logger = logging.getLogger('arrow-flight-module')
+
+def init_logger(loglevel_arg, color=False):
+    if color:
```

Comment:
I'm not sure we really need to concern ourselves with coloring log messages.  If the logger used includes coloring, great.  If not, then we don't need to implement it.   Please note, zerolog used in golang pkg/logging has two modes: one for pretty printing and one meant for efficient logging in production and when a machine is parsing the logs.  In the latter there is no coloring, etc.